include/cpu/kvm.h

```diff
@@ -330,8 +330,8 @@ struct kvm_run {
         char padding[1024];
     } s;
 
-    /* Shared timers state between QEMU and libcpu */
-    void *timers_state;
+    /* Share timer's clock scaling between QEMU and libcpu */
+    void *clock_scaling;
```

Comment:
Use an integer, not a pointer. When KVM_RUN exits, it will pass the updated scaling to the parent.

Reply:
Done. I didn't realise the kvm_run structure was periodically updated